Gemfile

```diff
@@ -66,6 +66,7 @@ group :development, :test do
   gem "factory_girl_rails"
   gem 'jasmine-rails'
   gem 'jasmine-jquery-rails'
+  gem 'pdf-reader', github: 'yob/pdf-reader'
```

Comment:
It's kind of strange that there's no recent release -- the project looks pretty active.